packages/account/src/Components/terms-of-use/terms-of-use.tsx

```diff
@@ -56,10 +77,7 @@ const TermsOfUse = ({
                                 is_disabled={isDesktop()}
                             >
                                 <ThemedScrollbars>
-                                    <div
-                                        className={cn('details-form__elements', 'terms-of-use')}
-                                        style={{ paddingBottom: isDesktop() ? 'unset' : null }}
-                                    >
+                                    <div className={cn('details-form__elements', 'terms-of-use')}>
```

Comment:
https://github.com/likhith-deriv/deriv-app/pull/45/files#r1278917449

Reply:
unset or null does the same as there is no override available

Reply:
can we change this classnames import to commonly used in our app? :)